content/en/docs/concepts/services-networking/service.md

```diff
@@ -608,14 +608,22 @@ By default, for LoadBalancer type of Services, when there is more than one port
 ports must have the same protocol, and the protocol must be one which is supported
 by the cloud provider.
 
-If the feature gate `MixedProtocolLBService` is enabled for the kube-apiserver it is allowed to use different protocols when there is more than one port defined. 
+If the feature gate `MixedProtocolLBService` is enabled for the kube-apiserver it is allowed to use different protocols when there is more than one port defined.
 
 {{< note >}}
 
 The set of protocols that can be used for LoadBalancer type of Services is still defined by the cloud provider.
 
 {{< /note >}}
 
+#### Disabling Load Balancer Node Ports
```

Comment:
(nit) headings should be sentence case:
```suggestion
#### Disabling load balancer NodePort allocation {#load-balancer-nodeport-allocation}
```

The custom fragment identifier is there so the heading persists even if the default changes in some future release.

Reply:
ack, will update